core/src/tile/tileManager.cpp

```diff
@@ -369,6 +369,8 @@ bool TileManager::addTile(TileSet& tileSet, const TileID& _tileID) {
     if (tile) {
         m_tiles.push_back(tile);
 
+        // Update tile origin based on wrap (set in the new tileID)
+        tile->updateTileOrigin(_tileID.wrap);
```

Comment:
Is this call necessary? I don't see any effect when I comment it. It seems strange that this function would even be public; it seems that a `Tile` could potentially get an origin that doesn't correspond to the `wrap` value in its `TileID`.